pkg/console/operator/operator.go

```diff
@@ -251,3 +260,36 @@ func (c *consoleOperator) removeConsole(cr *operatorsv1.Console) error {
 
 	return utilerrors.FilterOut(utilerrors.NewAggregate(errs), errors.IsNotFound)
 }
+
```

Comment:
We have a number of `ConfigMaps`, this does not handle syncing them all.

I would probably name this `SyncCustomLogoConfigMap()` . Take a look in `sync_v400()`, see where we call `SyncConfigMap()` and `SyncServiceCAConfigMap()`.  Generally, if you can follow that pattern for where this func lives, that will prob keep us maintainable / less messy.

Reply:
(This means you would call it as `func SyncCustomLogoConfigMap(c *consoleOperator, operatorConfig *operatorsv1.Console)`), but I do think that `sync_v400` ought to have been a method on the `operator` to match what you have.

Reply:
Why are you suggestion to pass `sync *resourceSyncer` into this?  I already have access to it through the operator.

Reply:
I see your latest change, that will work fine.